src/main/resources/application.properties

```diff
@@ -141,16 +141,16 @@ sunbird.cloud.media.storage.container=${SUNBIRD_CLOUD_MEDIA_STORAGE_CONTAINER:#{
 
 
 #doubtnut url
-doubtnut.baseurl=${DOUBTNUT_BASE_URL}
-doubtnut.apikey=${DOUBTNUT_AUTH_KEY}
-doubtnut.welcome.msg=${DOUBTNUT_WELCOME_MSG}
-doubtnut.welcome.video=${DOUBTNUT_WELCOME_VIDEO}
+doubtnut.baseurl=${DOUBTNUT_BASE_URL:#{""}}
+doubtnut.apikey=${DOUBTNUT_AUTH_KEY:#{""}}
+doubtnut.welcome.msg=${DOUBTNUT_WELCOME_MSG:#{""}}
```

Comment:
Add a dummy welcome message, so in case a msg does not exists, we can have something which is default